github.sgnl.yaml

```diff
@@ -0,0 +1,1099 @@
+        type: Bool
+      - name: isTemplate
+        externalId: $.node.isTemplate
+        type: Bool
+      - name: isUserConfigurationRepository
+        externalId: $.node.isUserConfigurationRepository
+        type: Bool
+      - name: mergeCommitAllowed
+        externalId: $.node.mergeCommitAllowed
+        type: Bool
+      - name: mirrorUrl
+        externalId: $.node.mirrorUrl
+        type: String
+      - name: name
+        externalId: $.node.name
+        type: String
+      - name: nameWithOwner
+        externalId: $.node.nameWithOwner
+        type: String
+      - name: openGraphImageUrl
+        externalId: $.node.openGraphImageUrl
+        type: String
+      - name: projectsResourcePath
+        externalId: $.node.projectsResourcePath
+        type: String
+      - name: projectsUrl
+        externalId: $.node.projectsUrl
+        type: String
+      - name: pushedAt
+        externalId: $.node.pushedAt
+        type: DateTime
+      - name: rebaseMergeAllowed
+        externalId: $.node.rebaseMergeAllowed
+        type: Bool
+      - name: resourcePath
+        externalId: $.node.resourcePath
+        type: String
+      - name: securityPolicyUrl
+        externalId: $.node.securityPolicyUrl
+        type: String
+      - name: shortDescriptionHTML
+        externalId: $.node.shortDescriptionHTML
+        type: String
+      - name: squashMergeAllowed
+        externalId: $.node.squashMergeAllowed
+        type: Bool
+      - name: stargazerCount
+        externalId: $.node.stargazerCount
+        type: Int64
+      - name: tempCloneToken
+        externalId: $.node.tempCloneToken
+        type: String
+      - name: updatedAt
+        externalId: $.node.updatedAt
+        type: DateTime
+      - name: url
+        externalId: $.node.url
+        type: String
+      - name: usesCustomOpenGraphImage
+        externalId: $.node.usesCustomOpenGraphImage
+        type: Bool
+      - name: viewerCanAdminister
+        externalId: $.node.viewerCanAdminister
+        type: Bool
+      - name: viewerCanCreateProjects
+        externalId: $.node.viewerCanCreateProjects
+        type: Bool
+      - name: viewerCanSubscribe
+        externalId: $.node.viewerCanSubscribe
+        type: Bool
+      - name: viewerCanUpdateTopics
+        externalId: $.node.viewerCanUpdateTopics
+        type: Bool
+      - name: viewerDefaultCommitEmail
+        externalId: $.node.viewerDefaultCommitEmail
+        type: String
+      - name: viewerHasStarred
+        externalId: $.node.viewerHasStarred
+        type: Bool
+      - name: viewerPossibleCommitEmails
+        externalId: $.node.viewerPossibleCommitEmails
+        type: String
+      - name: webCommitSignoffRequired
+        externalId: $.node.webCommitSignoffRequired
+        type: Bool
+  # https://docs.github.com/en/graphql/reference/objects#repository
+  Repository:
+    displayName: Repository
+    externalId: Repository
+    description: Repository entity in GitHub
```

Comment:
Update description based on the format decided above.

Reply:
fixed